test/unit/spec/signaling/v2/room.js

```diff
@@ -1901,7 +1919,12 @@ function makePeerConnectionManager(getRoom) {
 
     return new Map([
       ['foo', {
-        activeIceCandidatePair: { baz: 'zee' },
+        activeIceCandidatePair: {
+          baz: 'zee',
```

Comment:
If I understand correctly in your test you are relying that the peerConnection `foo` returns activeCandidate Pair stat with localCnadidate.relayProtocol set and peerConnection `bar` returns activeCandidatepair w/o ` localCnadidate.relayProtocol`

I think it's not very obvious from the `expectedArgs`parameters that you added.   Maybe it would help if you renamed `baz` - which seems like just a placeholder property to something more meaningful like
change:
``` 
activeIceCandidatePair: { baz: 'zee' },
```
to:
```
activeIceCandidatePair: { includesRelayProtocol: 'yes' },

```

 similarly you can change
```
activeIceCandidatePair: { zee: 'foo' },
```

```
activeIceCandidatePair: { includesRelayProtocol: 'no' },
```

That way you can use this `includesRelayProtocol` in your if statement instead of adding extra parameter to `expectedArgs `